src/hotspot/share/opto/intrinsicnode.cpp

```diff
@@ -368,3 +369,22 @@ const Type* ExpandBitsNode::Value(PhaseGVN* phase) const {
 
   return bitshuffle_value(src_type, mask_type, Op_ExpandBits, bt);
 }
+
+Node* ScopedValueGetLoadFromCacheNode::scoped_value() const {
+  Node* hits_in_cache = in(1);
+  assert(hits_in_cache->Opcode() == Op_ScopedValueGetHitsInCache, "");
+  return ((ScopedValueGetHitsInCacheNode*)hits_in_cache)->scoped_value();
```

Comment:
Why not add the neccessary bits to the class so you can use `as_ScopedValueGetLoadFromCache()`?